spec/swagger/definitions/court_hearing.json

```diff
@@ -48,7 +61,6 @@
           "case_type": {
             "type": "string",
             "example": "Adult",
-            "enum": ["Adult"],
```

Comment:
Is this the only change in this PR, apart the formatting?

Reply:
Yeah, it's used to generate the swagger doc that @teneightfive is validating against.